reference_validator/YAML_HotValidator.py

```diff
@@ -0,0 +1,724 @@
+#!/usr/bin/env python
+#coding=utf-8
+
+# File: YAML_HotValidator.py
+# Brief: Contains class YAML_HotValidator for validating references
+# Author: Katerina Pilatova (kpilatov)
+# Date: 2016
+
+from __future__ import with_statement, print_function
+
+import os
+import pprint
+import sys
+import six  # compatibility
+import yaml # pip install pyyaml
+
+import YAML_Enums as ENUM
+import YAML_Hotfile as HOT
+import YAML_HotClasses
+
+# only for Python 2.x
+if sys.version_info[0] == 2:
```

Comment:
This makes nyanbar a hard dependency on Python 2 even when the user opts out. Can you switch to this instead?

```python
try:
    import nyanbar
except ImportError:
    pass
```

That will work on Py3 once nyanbar is available there as well as make the program running without it installed.